temporalio/worker.py

```diff
@@ -0,0 +1,1112 @@
+"""Worker for processing Temporal workflows and/or activities."""
```

Comment:
It is very likely when workflows come on board, the activity code here will be split off into another file (the entire worker package will likely be a dir instead of a single file).